test/TestBase.py

```diff
@@ -167,17 +175,22 @@ def remove(self, items: typing.Union[Item, typing.Iterable[Item]]) -> None:
             self.multiworld.state.remove(item)
 
     def can_reach_location(self, location: str) -> bool:
+        """Determines if the current state can reach the provide location name"""
         return self.multiworld.state.can_reach(location, "Location", 1)
 
     def can_reach_entrance(self, entrance: str) -> bool:
+        """Determines if the current state can reach the provided entrance name"""
         return self.multiworld.state.can_reach(entrance, "Entrance", 1)
 
     def count(self, item_name: str) -> int:
+        """Returns the amount of an item currently in state"""
         return self.multiworld.state.count(item_name, 1)
 
     def assertAccessDependency(self,
                                locations: typing.List[str],
                                possible_items: typing.Iterable[typing.Iterable[str]]) -> None:
+        """Asserts that the provided locations can't be reached without all provided items but can be reached with any
```

Comment:
"all" sounds wrong here. "any" would probably be correct, but still sounds confusing :S
```suggestion
        """Asserts that the provided locations can't be reached without the listed items but can be reached with any
```